ogr/abstract.py

```diff
@@ -1,6 +1,7 @@
 import datetime
 from enum import Enum
 from typing import Optional, Match, List, Dict
+from urllib.request import urlopen
```

Comment:
I would ask that why not requests, but the code seems fairly easy

Reply:
...PyCharm auto imports...